### ISSUE TITLE
Retain same format as input and output for CSV

### ISSUE THREAD

**Issue by awsp** [uniVocity/univocity-parsers#191]
When I try to write rows to a new CSV, I could not reproduce the same result as it was from original text. 
I have searched the issue and found `setParseUnescapedQuotesUntilDelimiter()` although it is now deprecated, but I could not get the exact same result from input. 

```java        
    CsvParserSettings parseSettings = new CsvParserSettings();
    CsvParser parse = new CsvParser(parseSettings);
    //Original: "bbb","",,10,"15","aaa"
    List<String[]> records = parse.parseAll(new StringReader("\"bbb\",\"\",,10,\"15\",\"aaa\""));

    CsvWriterSettings writerSettings = new CsvWriterSettings();
    CsvWriter csvWriter = new CsvWriter(new File("/some/path/to/bbb.csv"), writerSettings);
    for (String[] record : records) {
        csvWriter.writeRow(record);
    }
    csvWriter.close();
```
Original:
"bbb","",,10,"15","aaa"

Result: 
bbb,,,10,15,aaa

Result: (With setQuoteAllFields() set to true)
"bbb","","","10","15","aaa"

I understand both CSV formats are correct, but I would like to retain the format as the input.
Is it possible to retain the column format as well? (Maybe using annotations? )

**Comment by jbax** [comment]
Currently you can't choose which columns will get quoted (or not) when writing. Will implement this in the next version.

**Comment by awsp** [comment]
I see. 
The closest setting I can see is actually the `setQuotationTriggers(char... triggers)`.
Looking forward to this new feature. Thanks!

**Comment by jbax** [comment]
Done, I've released a 2.6.0-SNAPSHOT build with this enhancement.

You can now use `writerSettings.quoteFields("C", "D");` or `writerSettings.quoteIndexes(0, 1, 4, 5);`

Thank you using our parsers!

**Comment by awsp** [comment]
Nice and neat! 
https://github.com/uniVocity/univocity-parsers/commit/e3bc7d025ae00675a9efb999b430069139954adc#diff-84303ba8a655a0dab7c1eb53267324f9R48

Thank you very much!
